### PR TITLE
test(arm support): Created new arm longevities

### PR DESCRIPTION
**Pull request by ShlomiBalalis**
Since arm is only supported in specific availability zones across different
aws regions, I set the availability_zone of all arm jobs to 'c' by default,
since it is supported both in us-east-1 and eu-west-1.

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: jenkins-pipelines/longevity-100gb-4h-arm.jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+longevityPipeline(
+    backend: 'aws',
+    region: 'eu-west-1',
+    test_name: 'longevity_test.LongevityTest.test_custom_time',
+    test_config: '''["test-cases/longevity/longevity-100gb-4h.yaml", "configurations/arm_instance_types/Is4gen_4xlarge.yaml"]''',
+    availability_zone: 'c',
```
Comment (fgelcer) [inline jenkins-pipelines/longevity-100gb-4h-arm.jenkinsfile:11]: is it true to all the regions we support?

Comment (ShlomiBalalis) [reply jenkins-pipelines/longevity-100gb-4h-arm.jenkinsfile:11]: It's try for every region that supports ARM at all

Comment (ShlomiBalalis) [reply jenkins-pipelines/longevity-100gb-4h-arm.jenkinsfile:11]: (That my account has access to. I can't talk for `us-gov-west-1`)

Comment (ShlomiBalalis) [reply jenkins-pipelines/longevity-100gb-4h-arm.jenkinsfile:11]: In some of them, arm doesn't seem to be supported at all, so obviously there's nothing we can do about them.
Of those that do support them, some support regions a-c while other b-d

**Review by fruch — APPROVED** [review]
LGTM

**Review by fgelcer — APPROVED** [review]
LGTM

File: configurations/arm_instance_types/Is4gen_4xlarge.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+instance_type_db: 'is4gen.4xlarge'  # The equivalent of i3.4xlarge
+# enhanced network isn't supported
+append_scylla_setup_args: ' --no-ec2-check '
+use_mgmt: false
```
Comment (roydahan) [inline configurations/arm_instance_types/Is4gen_4xlarge.yaml:4]: Someone said that manager-2.6 supports arm as well, isn't it true?

File: configurations/arm_instance_types/Is4gen_4xlarge.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+instance_type_db: 'is4gen.4xlarge'  # The equivalent of i3.4xlarge
```
Comment (roydahan) [inline configurations/arm_instance_types/Is4gen_4xlarge.yaml:1]: This is incorrect!
the instance type that eqvivealnt to i3.4xlarge is **im4gn.4xlarge**.
Please edit comment, add another configuration and change all longevities below to use that configuration.

With Is4gen configuration, configure a longevity that is equivalent to **i3en**.

**Comment by roydahan** [comment]
@ShlomiBalalis please send a new PR to address my comments above.
The change of the AZ in the pipeline should've been a separate commit.

@fruch you're light with your finger on the merge button, too quick for me to review.

**Pull request merged**